data/json/items/ammo/shot.json

```diff
@@ -132,159 +93,108 @@
     "count" : 5,
-    "stack_size" : 20,
-    "ammo_type" : "shot",
-    "casing" : "shot_hull",
     "damage" : 10,
-    "range" : 12,
-    "dispersion" : 90,
-    "recoil" : 300,
-    "loudness": 140,
-    "effects" : ["COOKOFF", "EXPLOSIVE_SMALL"]
+    "pierce": 0,
+    "extend": { "effects": [ "EXPLOSIVE_SMALL" ] }
   },
   {
     "id" : "shot_scrap",
+    "copy-from": "shot_00",
     "type" : "AMMO",
     "name" : "makeshift shotgun shot",
+    "name_plural": "makeshift shotgun shot",
     "description" : "A shotgun shell filled with whatever was lying around.  They are more damaging than birdshot, but fairly inaccurate.",
-    "weight" : 32,
-    "volume" : 1,
-    "price" : 1800,
-    "count" : 20,
-    "stack_size" : 20,
-    "material" : ["plastic", "powder"],
-    "symbol" : "=",
-    "color" : "red",
-    "ammo_type" : "shot",
-    "casing" : "shot_hull",
-    "damage" : 25,
-    "range" : 4,
-    "dispersion" : 2,
-    "recoil" : 30,
-    "loudness": 160,
-    "effects" : ["COOKOFF", "SHOT", "RECYCLED"]
+    "proportional": {
+        "price": 0.4,
+        "damage": 0.6,
+        "dispersion": 1.2
+    },
+    "extend": { "effects": [ "RECYCLED" ] },
+    "delete": { "effects": [ "NEVER_MISFIRES" ] }
   },
   { 
     "id" : "shot_scrapslug",
+    "copy-from": "shot_slug",
     "type" : "AMMO",
     "name" : "makeshift shotgun slug",
+    "name_plural": "makeshift shotgun slug",
     "description" : "A shotgun shell filled with a slug made from whatever that was lying around.  It is fairly inaccurate, and does little damage, but it works in a pinch.",
-    "weight" : 32,
-    "volume" : 1,
-    "price" : 1800,
-    "count" : 20,
-    "stack_size" : 20,
-    "material" : ["plastic", "powder"],
-    "symbol" : "=",
-    "color" : "red",
-    "ammo_type" : "shot",
-    "casing" : "shot_hull",
-    "damage" : 15,
-    "pierce" : 3,
-    "range" : 8,
-    "dispersion" : 6,
-    "recoil" : 28,
-    "loudness": 160,
-    "effects" : ["COOKOFF", "RECYCLED"]
+    "proportional": {
+        "price": 0.4,
+        "damage": 0.6,
+        "pierce": 0.6,
+        "range": 0.8,
+        "dispersion": 1.2
+    },
+    "extend": { "effects": [ "RECYCLED" ] },
+    "delete": { "effects": [ "NEVER_MISFIRES" ] }
   },
   {
     "id" : "shot_slug",
+    "copy-from": "shot_00",
     "type" : "AMMO",
     "name" : "shotgun slug",
     "description" : "A heavy metal slug used with shotguns to give them the range capabilities of a rifle.  Extremely damaging but rather inaccurate.",
-    "material" : ["plastic", "powder"],
-    "symbol" : "=",
-    "color" : "red",
-    "weight" : 32,
-    "volume" : 1,
     "price" : 4500,
-    "count" : 20,
-    "stack_size" : 20,
-    "ammo_type" : "shot",
-    "casing" : "shot_hull",
-    "damage" : 50,
-    "pierce" : 8,
-    "range" : 12,
-    "dispersion" : 60,
-    "recoil" : 420,
-    "loudness": 160,
-    "effects" : ["COOKOFF"]
+    "relative": {
+        "pierce": 8,
+        "range": 6,
+        "recoil": 30
```

Comment:
That would make the slug version almost strictly better than the shot.

Should probably lose damage and gain dispersion. Shot's low dispersion is supposed to represent the spread.